LibraryComponents/Classes/Core/AndesMessage/Bullets/AndesBullet.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Bullet.swift
+//  AndesUI
+//
+//  Created by Martin Ezequiel Becerra on 2/25/21.
+//
+
+import Foundation
+
+/// Class to create Body Bullets for an AndesMessage
+@objc public final class AndesBullet: NSObject {
+    let text: String
+    var bodyLinks: AndesBodyLinks?
```

Comment:
```suggestion
    let bodyLinks: AndesBodyLinks?
```

Es una sugerencia, implicar hacer un if en el init.